include/seqan3/alignment/scoring/detail/simd_match_mismatch_scoring_scheme.hpp

```diff
@@ -150,6 +150,15 @@ class simd_match_mismatch_scoring_scheme
     }
     //!\}
 
+    /*!\brief Returns the match score used for padded symbols.
+     *
+     * \returns The score used for aligning two padding symbols.
```

Comment:
:nail_care: Do you need this `\returns` documentation? It's just stating the same as the brief so you might as well omit it